lib/code_corps/services/project.ex

```diff
@@ -12,8 +12,7 @@ defmodule CodeCorps.Services.ProjectService do
       StripeConnectSubscription
       |> where([s], s.status == "active" and s.stripe_connect_plan_id == ^plan_id)
       |> Repo.aggregate(:sum, :quantity)
-
-    total_monthly_donated = default_to_zero(total_monthly_donated)
+      |> default_to_zero
```

Comment:
I wrote this and now, looking at this, I have no idea where my brain was. Probably lack of (or too much) coffee. Nice find and good fix! 👍